docs/install.rst

```diff
@@ -1,26 +1,126 @@
+
+        `aarch64 <https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-aarch64.sh>`__
+
+        `ppc64le <https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-ppc64le.sh>`__
+
+    .. grid-item-card:: Windows
+        :link: https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Windows-x86_64.exe
+
+        `x86-64 <https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Windows-x86_64.exe>`__
+
+    .. grid-item-card:: Mac
+
+        `x86-64 <https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-MacOSX-x86_64.sh>`__
+
+        `arm64 (Apple Silicon) <https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-MacOSX-arm64.sh>`__
+
+Then select your platform to install miniforge:
+
+.. tab-set::
+
+    .. tab-item:: Linux & Mac
+        :sync: platform
+
+        Linux & Mac Run the script downloaded above, with
+        ``bash <filename>``. The following should work:
+
+        .. code-block:: console
+
+            bash Miniforge3-$(uname)-$(uname -m).sh
+
+        Once the installer has completed, close and reopen your terminal.
+
+    .. tab-item:: Windows
+        :sync: platform
+
+        Double click the executable file downloaded from
+        the links above.
+
+        Once the installer has completed you should have a new "miniforge
+        Prompt" entry in your start menu.
+
+In a new terminal (miniforge Prompt on Windows) run ``conda list`` to test that the install has worked.
+
+Using an Existing Conda Installation
+====================================
+
+If you already have a conda installation you can use it to install astropy.
+We do not recommend reusing an anaconda installation, as the default packages provided by Anaconda Inc. are often out dated and can require a license for use at a large organisaiton.
+If you have installed miniforge or anaconda you can add ``-c conda-forge`` to all installation commands to install packages from `conda-forge <https://conda-forge.org>`__.
+
+
+Installing astropy
+==================
+
+To install ``astropy``, start by launching a terminal (under a UNIX-like system) or the miniforge Prompt (under Windows).
+Now we will create and activate a new virtual environment to install ``astropy`` into:
+
+.. code-block:: bash
+
+    $ conda create --name astropy
+    $ conda activate astropy
+
+In this case the environment we have created is named 'astropy'.
+Feel free to change this to a different environment name.
+
+The benefit of using a virtual environment is that it allows you to install packages without affecting any other Python installation on your system.
+This also means you can work on multiple projects (research or coding) with different package requirements without them interfering with each other.
+
+Now we have a fresh environment we can install ``astropy``:
+
+.. code-block:: bash
+
+    $ conda install astropy
```

Comment:
At this point I think we should recommend the `pip install` path for installing astropy. In other words `conda` is great for creating the base Python isolated (or "virtual") environment, but pip works better for installing the rest. In particular it is much easier to specify additional dependencies `recommended` or `all`. `conda install astropy` installs only minimal dependencies and there is no easy way to get the recommended or all dependencies.

Instead, start with:
```
conda create -n astropy-env python
```
Then say "Now install astropy using pip as described in the next section."

Reply:
fwiw, i completely agree with this and have recommended this approach in other contexts for the stated reasons. conda is great for pulling in the proverbial kitchen sink, but if you want to fine-tune what you install, pip is a much better approach.